### PR TITLE
Site copy - May 1 updates

### PR DESCRIPTION
**Pull request by hcline**
- Updated home page copy
- Updated case study content

### CODE REVIEW

**Comment by jayperryworks** [comment]
@hcline if you can update from master, that should pull in the new design thinking page. I can add any buttons pointing there/update as needed.

**Comment by hcline** [comment]
@jayperryworks hopefully I updated correctly!

File: data/site.yml [hunk 1]
```diff
@@ -20,19 +20,18 @@ staff:
 # -> primarily used in the footer
 contact:
   # Title for contact area in footer, e.g. 'You used to call me on my cell phone'
-  headline: Contact us
+  headline: Want to do something differently?
   # A short blurb - just one paragraph, please.
-  intro: We’d love to hear about your ideas and the problems you want to solve. Contact us, and let’s get the conversation started.
```
Comment (jayperryworks) [inline data/site.yml]: Just a thought -- the previous intro was maybe superfluous but it does seem helpful to have an intro blurb of some kind here, if nothing else just to emphasize that we have international locations/staff.

Also I'm not totally sure about "Want to do something differently" as the headline -- it feels like the 'differently' theme is getting a bit repetitive, and it seems a bit less clear what we want users to do here. Maybe something along the lines of "Ready to get started?" -- really like the question scheme but feel like we need to be more explicit that you're going to contact us and there are several ways/places to reach us.

File: data/case_studies.yml [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+  home: How do we do this?
   # the headline for the 'more work' nav at the bottom of each case study
   nav: More work
 
 # an intro blurb describing our work
 intro: |-
-  <p>By applying design thinking to a bunch of situations (from relatively simple to super complex), across diverse sectors (healthcare, learning, disaster management, public-private partnerships, consumer products), for diverse needs (technology, innovation, knowledge management, branding, communications) we have helped our clients (NGOs, corporates, international development organizations) to achieve better outcomes than they thought possible.</p>
-  <p>Here's a taste of what we've done:</p>
+  <p>We’re specialists in strategy, branding, design, communications, knowledge management, and technology. We use these skills in our consulting engagements and the development of new products and applications.</p>
+  <p>We’ve taken this approach on projects in development, healthcare, and education, among others, and with a wide variety of partners, from international development organisations to nonprofits and private sector firms. </p>
```
Comment (jayperryworks) [inline data/case_studies.yml]: This is great - feels much more succinct/clear than before. Only suggestion is maybe add a sentence at the end to make it clear that what you're seeing below are case studies? e.g. "Here's some examples of recent work:"

**Pull request merged**